From 7562ffbce9d768d5af9d23361cf6dd6ef992bead Mon Sep 17 00:00:00 2001
From: Jiffin Tony Thottan <jthottan@redhat.com>
Date: Fri, 10 Nov 2017 23:38:14 +0530
Subject: [PATCH 29/74] build: exclude glusterfssharedstorage.service and
 mount-shared-storage.sh from client builds

Label: DOWNSTREAM ONLY

Change-Id: I7d76ba0742b5c6a44505eb883eacda0c91efbe51
Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/109684
Reviewed-by: Milind Changire <mchangir@redhat.com>
Tested-by: Milind Changire <mchangir@redhat.com>
Reviewed-by: Atin Mukherjee <amukherj@redhat.com>
---
 glusterfs.spec.in | 22 +++++++++++++++++++++-
 1 file changed, 21 insertions(+), 1 deletion(-)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index 343e88f..4596e3f 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -1083,6 +1083,20 @@ exit 0
 %exclude %{_libexecdir}/glusterfs/glusterfind
 %exclude %{_bindir}/glusterfind
 %exclude %{_libexecdir}/glusterfs/peer_add_secret_pub
+# exclude eventsapi files
+%exclude %{_sysconfdir}/glusterfs/eventsconfig.json
+%exclude %{_sharedstatedir}/glusterd/events
+%exclude %{_libexecdir}/glusterfs/events
+%exclude %{_libexecdir}/glusterfs/peer_eventsapi.py*
+%exclude %{_sbindir}/glustereventsd
+%exclude %{_sbindir}/gluster-eventsapi
+%exclude %{_datadir}/glusterfs/scripts/eventsdash.py*
+%if ( 0%{?_with_systemd:1} )
+%exclude %{_unitdir}/glustereventsd.service
+%exclude %_init_glusterfssharedstorage
+%else
+%exclude %{_sysconfdir}/init.d/glustereventsd
+%endif
 # exclude server files
 %exclude %{_sharedstatedir}/glusterd/*
 %exclude %{_sysconfdir}/glusterfs
@@ -1123,6 +1137,9 @@ exit 0
 %exclude %{_sbindir}/glusterd
 %exclude %{_sbindir}/snap_scheduler.py
 %exclude %{_datadir}/glusterfs/scripts/stop-all-gluster-processes.sh
+%if ( 0%{?_with_systemd:1} )
+%exclude %{_libexecdir}/glusterfs/mount-shared-storage.sh
+%endif
 %exclude %{_sbindir}/conf.py*
 %if 0%{?_tmpfilesdir:1}
 %exclude %{_tmpfilesdir}/gluster.conf
@@ -2181,7 +2198,10 @@ end
 * Thu Jul 13 2017 Kaleb S. KEITHLEY <kkeithle@redhat.com>
 - various directories not owned by any package
 
-* Fri Jun 16 2017 Jiffin Tony Thottan <jthottan@redhat.com>
+* Wed Jun 21 2017 Jiffin Tony Thottan <jthottan@redhat.com>
+- Exclude glusterfssharedstorage.service and mount-shared-storage.sh from client builds
+
+* Tue Jun 20 2017 Jiffin Tony Thottan <jthottan@redhat.com>
 - Add glusterfssharedstorage.service systemd file
 
 * Fri Jun 9 2017 Poornima G <pgurusid@redhat.com>
-- 
1.8.3.1

